src/vision.cs

```diff
@@ -1725,10 +1843,6 @@ interface VNImageHomographicAlignmentObservation {
 		Matrix3 WarpTransform {
 			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
 			get;
-#if !XAMCORE_4_0
-			[NotImplemented]
-			set;
-#endif
```

Comment:
why ? that's a breaking change to remove a setter

Reply:
it remains a breaking change for managed code
I suspect we detected (crash?) this before and that's why this code is already under `#if !XAMCORE_4_0` and throwing a NIE